Source/WebCore/platform/audio/PlatformMediaSessionManager.h

```diff
@@ -238,6 +240,9 @@ class PlatformMediaSessionManager
 #if ENABLE(ALTERNATE_WEBM_PLAYER)
     static bool m_alternateWebMPlayerEnabled;
 #endif
+#if HAVE(SC_CONTENT_SHARING_PICKER)
+    static bool m_useSCContentSharingPicker;
```

Comment:
Not new to this patch but it is a little weird to use `m_` prefix for static members. I think we often use `s_` prefix for those.

Reply:
OK, changed.